NCEP/gc_datadissm_ursa.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash --login
+
+#SBATCH --ntasks=1
+#SBATCH --account=nems
+#SBATCH --partition=u1-service
+#SBATCH --time=10:00
+#SBATCH --job-name=eagle-solo
+#SBATCH --output=slurm/gcgfs_datadissm.out
+#SBATCH --error=slurm/gcgfs_datadissm.err
+
+# load modules
+module use /contrib/spack-stack/spack-stack-1.9.1/envs/ue-oneapi-2024.2.1/install/modulefiles/Core/
+module load stack-oneapi
+module load awscli-v2/2.15.53
+module list
+
+curr_datetime=$1
+num_pressure_levels=13
+COMROOT=/scratch3/NCEPDEV/stmp/Linlin.Cui/ptmp
```

Comment:
Can this COMROOT be passed in? e.g. COMROOT=${2:-/scratch3/NCEPDEV/stmp/$USER/ptmp}